client/batch.go

```diff
@@ -308,6 +315,29 @@ func (b *Batch) Scan(s, e interface{}, maxRows int64) {
 	b.initResult(1, 0, nil)
 }
 
+// ReverseScan retrieves the rows in reserve order starting at end (exclusive)
+// and finishing at begin(inclusive).
+//
+// A new result will be appended to the batch which will contain up to maxRows
+// rows and Result.Err will indicate success or failure.
+//
+// key can be either a byte slice, a string, a fmt.Stringer or an
+// encoding.BinaryMarshaler.
+func (b *Batch) ReverseScan(s, e interface{}, maxRows int64) {
```

Comment:
> I think it is better to change the semantics of reverse scan to start key exclusive and end key inclusive to client interface

I'm getting confused with all the keys here. Do you mean `("a","b")` (left and right side excluded)? That's what I was suggesting. Scanning `("a", "b"]` (left-exclude, right-include) is just calling `Next()` on the right key.